recipe/meta.yaml

```diff
@@ -5,51 +5,51 @@ package:
   version: {{ version }}
 
 source:
-  fn: ibis-framework-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/i/ibis-framework/ibis-framework-{{ version }}.tar.gz
-  md5: b452f718ab8c607b01405e8ee40abf86
+  git_rev: {{ "v" + version }}
+  git_url: https://github.com/ibis-project/ibis
 
 build:
-  number: 1
+  number: 2
   script: python setup.py install --single-version-externally-managed --record record.txt
 
 requirements:
   build:
     - enum34  # [py27]
-    - graphviz
-    - impyla >=0.14.0
+    - funcsigs  # [py27]
+    - functools32  # [py27]
     - multipledispatch
     - numpy >=1.10.0
     - pandas >=0.18.1
-    - psycopg2
     - python
-    - python-graphviz
-    - python-hdfs >=2.0.0  # [not py36]
+    - regex
     - setuptools
     - six
-    - sqlalchemy >=1.0.0
-    - thrift  # [py36]
-    - thrift <=0.9.3  # [not py36]
-    - thriftpy <=0.3.9
     - toolz
```

Comment:
Would encourage you to drop everything from `build` that is not needed at build time and re-render again.

Reply:
Ok great, will give that a try. Thank you.